release-notes/opensearch-release-notes-1.3.5.md

```diff
@@ -5,13 +5,6 @@
 OpenSearch and OpenSearch Dashboards 1.3.5 includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
```

Comment:
You may also remove the headings here since remove the ones below.

Reply:
Updated @zelinh.